swift/ios-application/build.gradle

```diff
@@ -0,0 +1,26 @@
+buildscript {
+    dependencies {
+        classpath "org.gradle.samples:ios-plugin:latest.integration"
+    }
+}
+
+plugins {
+    id 'swift-application'
+    id 'xcode'
+}
+
+apply plugin: 'org.gradle.samples.ios-application'
+
+group = "org.gradle.samples"
+
+// Add iOS specific resource to Xcode IDE project
+file('src/main/resources/Base.lproj').listFiles().each {
```

Comment:
should be something like `xcode.project.groups.sources.from fileTree('src/main/resources/Base.lproj')`, to avoid listing the files at configuration time.